bentoml/_internal/io_descriptors/base.py

```diff
@@ -7,6 +8,16 @@
 IOPyObj = t.TypeVar("IOPyObj")
 
 
+def _mk_repr(obj: t.Any) -> str:
```

Comment:
`__repr__` goal is to be unambiguous. Mostly you can eval a repr to create the same object.
`__str__` goal is to be readable.